src/org/openstreetmap/josm/actions/SelectSharedChildObjectsAction.java

```diff
@@ -0,0 +1,68 @@
+// License: GPL. For details, see LICENSE file.
+package org.openstreetmap.josm.actions;
+
+import static org.openstreetmap.josm.gui.help.HelpUtil.ht;
+import static org.openstreetmap.josm.tools.I18n.tr;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.KeyEvent;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.openstreetmap.josm.data.osm.IPrimitive;
+import org.openstreetmap.josm.data.osm.OsmPrimitive;
+import org.openstreetmap.josm.tools.Shortcut;
+import org.openstreetmap.josm.tools.Utils;
+
+/**
+ * Select child objects (way nodes and relation members) that are shared by all objects in the current selection.
+ */
+public class SelectSharedChildObjectsAction extends JosmAction {
+
+    /**
+     * Create a new SelectSharedChildObjectsAction
+     */
+    public SelectSharedChildObjectsAction() {
+        super(tr("Shared Child Objects"),
+                null,
+                tr("Select child objects (way nodes and relation members) that are shared by all objects in the current selection"),
+                Shortcut.registerShortcut("selection:sharedchildobjects",
+                    tr("Selection: {0}", tr("Shared Child Objects")),
+                    KeyEvent.CHAR_UNDEFINED, Shortcut.NONE),
```

Comment:
No default shortcut works for now. Especially since there aren't a lot of valid "easy" remaining keys for it.